v2/sourcedb-to-spanner/src/test/java/com/google/cloud/teleport/v2/templates/SourceDbToSpannerTest.java

```diff
@@ -48,6 +95,105 @@ public void testCreateSpannerConfig() {
     assertEquals(config.getDatabaseId().get(), "testDatabaseId");
   }
 
+  @Test
+  public void createIdentitySchemaMapper() {
+    SourceDbToSpannerOptions mockOptions = createOptionsHelper("", "");
+    ISchemaMapper schemaMapper = SourceDbToSpanner.getSchemaMapper(mockOptions, spannerDdl);
+    assertTrue(schemaMapper instanceof IdentityMapper);
+  }
+
+  @Test
+  public void createSessionSchemaMapper() {
+    SourceDbToSpannerOptions mockOptions =
+        createOptionsHelper(
+            Paths.get(Resources.getResource("session-file-with-dropped-column.json").getPath())
+                .toString(),
+            null);
+    ISchemaMapper schemaMapper = SourceDbToSpanner.getSchemaMapper(mockOptions, spannerDdl);
+    assertTrue(schemaMapper instanceof SessionBasedMapper);
+  }
+
+  @Test(expected = Exception.class)
```

Comment:
nit: Me and @VardhanThigle discussed over previous code reviews how it is better practice to use assertThrows over an expected clause. We should try that in the future also.